sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -87,6 +87,11 @@ export class ApplicationStudentsController extends BaseController {
     super();
   }
 
+  /**
+   * Get application details by id.
+   * @param id for the application to be retrieved.
+   * @returns application details.
+   */
   @Get(":id")
   @ApiOkResponse({
```

Comment:
I believe we decided not to use `@ApiOkResponse` but the time when we decided that as team it was already created in some places. I guess these ones are left overs like that.

The reason was that, if we use the ApiOkResponse the request body is not shown out of box in Swagger for Patch and Post. I assume there is difference in Get, but we aren't using the ApiOkResponse unless there is a specific reason.

![image](https://user-images.githubusercontent.com/54600590/211064198-ae50be65-8dc3-40ae-b1a1-3b91d9f20f9f.png)